R/get_gene_expression.R

```diff
@@ -1,57 +1,109 @@
+
+#' @title Check Gene Expression.
```

Comment:
Documentation is incomplete but I think this will really simplify the process and make it more flexible for the users moving forward